kedro_vertexai/context_helper.py

```diff
@@ -100,19 +101,19 @@ def _ensure_obj_is_dict(self, obj):
     def config(self) -> PluginConfig:
         cl: AbstractConfigLoader = self.context.config_loader
         try:
-            obj = self.context.config_loader.get(self.CONFIG_FILE_PATTERN)
-        except:  # noqa
-            obj = None
-
-        if obj is None:
-            try:
-                obj = self._ensure_obj_is_dict(
-                    self.context.config_loader[self.CONFIG_KEY]
+            if self.CONFIG_KEY not in cl.config_patterns.keys():
```

Comment:
Let's add a test for that, because right now unit tests are covering the case where the `config_patterns` **has** vertexai added.

https://github.com/getindata/kedro-vertexai/blob/0bcb35e6dbe4eb2b26c1499065ec68a077e8bb9f/tests/test_context_helper.py#L73